samples/packaged/CopilotChat/SKWebApi/Skills/ChatSkill.cs

```diff
@@ -302,7 +302,7 @@ public async Task SaveNewMessageAsync(string message, SKContext context)
     {
         var allMemories = this.GetAllMemoriesAsync(context);
 
-        return await allMemories.FirstAsync();
+        return await allMemories.FirstOrDefaultAsync();
```

Comment:
When there is no memory store, this would throw since there was no first memory.